### PR TITLE
Adds a chance for ID cards to be tastefully thick

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

Adds a 1% chance for any ID to have a new trait (100% on golden IDs)
This trait adds a special interaction with ID appraisal chips
https://streamable.com/pdu7kj

## Why It's Good For The Game

Impressive. Very nice. Let's see the Captain's card.

## Changelog

:cl:
add: Adds a chance for ID cards to be tastefully thick
/:cl:

### CODE REVIEW

File: code/game/objects/items/cards_ids.dm [hunk 1]
```diff
@@ -692,8 +694,28 @@
...
+				living_user.adjust_timed_status_effect(10 SECONDS, /datum/status_effect/jitter)
+			addtimer(CALLBACK(src, .proc/drop_card, user), 10 SECONDS)
 	. += span_notice("<i>There's more information below, you can look again to take a closer look...</i>")
 
+/obj/item/card/id/proc/drop_card(mob/user)
+	user.stop_sound_channel(CHANNEL_HEARTBEAT)
+	REMOVE_TRAIT(src, TRAIT_NODROP, "psycho")
+	if(user.is_holding(src))
+		user.dropItemToGround(src)
+	for(var/mob/living/carbon/human/viewing_mob in viewers(user, 7))
```
Comment (JohnFulpWillard) [inline code/game/objects/items/cards_ids.dm:713]:
```suggestion
	for(var/mob/living/carbon/human/viewing_mob in viewers(user, 2))
```

**Comment by Wallemations** [comment]
Will this interfere with looking twice for the extended description?

**Comment by ATH1909** [comment]
10 seconds seems a bit long; how long does he stare at the card in the actual scene?

File: code/game/objects/items/cards_ids.dm [hunk 2]
```diff
@@ -1006,6 +1028,10 @@
 	inhand_icon_state = "gold_id"
 	wildcard_slots = WILDCARD_LIMIT_GOLD
 
+/obj/item/card/id/advanced/gold/Initialize(mapload)
+	. = ..()
+	ADD_TRAIT(src, TRAIT_TASTEFULLY_THICK_ID_CARD, ROUNDSTART_TRAIT)
```
Comment (optimumtact) [inline code/game/objects/items/cards_ids.dm:1033]: this makes it impossible for them to handle gold cards

Comment (optimumtact) [reply code/game/objects/items/cards_ids.dm:1033]: no it's okay, I thought it also applied when holding it

**Pull request merged**